.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
@hedss I'm not convinced we should have a `.vscode` in the repositories of projects themselves, it seems awfully programmer specific. It makes sense to maybe have it in this repo as an example, but I'm not sure it should actually be added to typescript project repos.